webpack.config.js

```diff
@@ -66,8 +66,12 @@ module.exports = {
             }]
         },
         {
+            test: /\.(png|wav)$/,
+            loader: 'raw-loader'
+        },
+        {
             test: /\.svg$/,
-            loader: 'svg-url-loader?noquotes'
```

Comment:
This change is just to get the svg loader in line with what's standard. Using svg-url-loader in the first place was a mistake, since we want a loader that can handle more than just svgs.